contracts/Core/AssetManager.sol

```diff
@@ -31,9 +29,22 @@ contract AssetManager is ACL, AssetStorage {
         string selector,
         string name,
         bool repeat,
-        address creator,
```

Comment:
IMO creator is required since admin may change. Any specific reason this was removed?

Reply:
Removed it as the admin can only create jobs/collection but makes sense to bring it back as the admin might change.

Reply:
👍